ppr-ui/tests/unit/CautionBox.spec.ts

```diff
@@ -19,14 +19,14 @@ const store = useStore()
  *
  * @returns a Wrapper<SearchedResultPpr> object with the given parameters.
  */
-function createComponent (setMsg: string): Wrapper<any> {
+function createComponent (setMsg: string, setImportantWord?: string): Wrapper<any> {
```

Comment:
I'm not sure if its useful here but i made a quick common `createComponent` method in the test utils, perhaps see if that could* work here. 

Saves us from repeating this same method in every unit test file. 
Your call, just have a look :)

Reply:
Sounds good thanks for the heads up!